modules/dr-restoring-cluster-state.adoc

```diff
@@ -228,22 +260,22 @@ csr-zhhhp   3m8s   kubernetes.io/kube-apiserver-client-kubelet   system:servicea
 <1> A pending kubelet service CSR (for user-provisioned installations).
 <2> A pending `node-bootstrapper` CSR.
 
-.. Review the details of a CSR to verify that it is valid:
+.. Review the details of a CSR to verify that it is valid by running the following command:
```

Comment:
I do not see "CSR" expanded in the document. Maybe expand it's first occurence in the doc?